llvm/include/llvm/ExecutionEngine/JITLink/x86_64.h

```diff
@@ -95,6 +95,19 @@ enum EdgeKind_x86_64 : Edge::Kind {
   ///
   Delta32,
 
+  /// An 8-bit delta.
+  ///
+  /// Delta from the fixup to the target.
+  ///
+  /// Fixup expression:
+  ///   Fixup <- Target - Fixup + Addend : int64
```

Comment:
I just noticed that this comment should have `: int8` too.

Reply:
Makes sense. Fixed the comment for `Delta32` above as well.

Reply:
Oh nice catch, thank you!